SwrveConversationSDK/Conversation/SwrveConversationAtom.h

```diff
@@ -3,6 +3,16 @@
 
 @class SwrveBaseConversation;
 
+#define kSwrveKeyTag @"tag"
+#define kSwrveKeyType @"type"
+#define kSwrveKeyValues @"values"
+#define kSwrveKeyAnswerId @"answer_id"
+#define kSwrveKeyAnswerText @"answer_text"
+#define kSwrveKeyStyle @"style"
+#define kSwrveKeyFontFile @"font_file"
+#define kSwrveKeyTextSize @"text_size"
+#define kSwrveKeyAlignment @"alignment"
+
```

Comment:
Valid point but would prefer seeing this done as a new jira, so created one here:
https://swrvedev.jira.com/browse/SWRVE-14676